draft-ietf-quic-http.md

```diff
@@ -88,28 +88,100 @@ code and issues list for this draft can be found at
-{{!RFC7540}}.
-
-
-## Notational Conventions
+{{!HTTP2=RFC7540}}.
+
+# HTTP/3 Protocol Overview
+
+HTTP/3 provides a transport for HTTP semantics using the QUIC transport protocol
+and an internal framing layer similar to HTTP/2.
+
+An HTTP/3 endpoint is discovered using HTTP Alternative Services.  Once a client
+knows that an HTTP/3 server exists at a certain endpoint, it opens a QUIC
+connection. QUIC provides protocol negotiation, stream-based multiplexing, and
+flow control.
+
+Within each stream, the basic unit of HTTP/3 communication is a frame
+({{frames}}).  Each frame type serves a different purpose.  For example, HEADERS
+and DATA frames form the frames form the basis of HTTP requests and responses
+({{request-response}}).  Other frame types like SETTINGS, PRIORITY, and GOAWAY
+are used to managed the overall connection.
+
+Multiplexing of requests is performed using the QUIC stream abstraction, with
+each request and response consuming a single QUIC stream.  Streams are largely
+independent of each other, so one stream that is blocked or suffers packet loss
+does not prevent progress on other streams.
+
+Server push is an interaction mode introduced in HTTP/2 {{!HTTP2}} which permits
+a server to push a request-response exchange to a client in anticipation of the
+client making the indicated request.  This trades off network usage against a
+potential latency gain.  Several HTTP/3 frames are used to manage server push,
+such as PUSH_PROMISE, DUPLICATE_PUSH, and CANCEL_PUSH.
+
+As in HTTP/2, request and response headers are compressed for transmission.
+Because HPACK {{?HPACK=RFC7231}} relies on in-order transmission of compressed
+header blocks (a guarantee not provided by QUIC), HTTP/3 replaces HPACK with
+QPACK [QPACK].  QPACK uses separate unidirectional streams to modify and track
+header table state, while header blocks refer to the state of the table without
+modifying it.
+
+## Document Organization
+
+The HTTP/3 specification is split into seven parts:
+
+- Connection Setup and Management ({{connection-setup}}) covers how an HTTP/3
+  endpoint is discovered and a connection is established.
+- Stream Mapping and Usage ({{stream-mapping}}) describes the way QUIC streams
+  are used.
+- HTTP Framing Layer ({{http-framing-layer}}) describes the frames used on
+  most streams.
+- HTTP Request Lifecycle ({{http-request-lifecycle}}) describes how HTTP
+  semantics are expressed using frames.
+- Connection Closure ({{connection-closure}}) describes how connections are
+  terminated, either gracefully or abruptly.
+- Extensions to HTTP/3 ({{extensions}}) describes how new capabilities can be
+  added in future documents.
+- Error Handling ({{errors}}) describes how error conditions are handled and
+  expressed, either on a particular stream or for the connection as a whole.
+
+Readers familiar with HTTP/2 will find a more detailed comparison with that
+protocol in {{h2-considerations}}.
```

Comment:
With that protocol? HTTP/1.1?

Reply:
@mikkelfj : I suspect you're missing that comparison "of HTTP/3" is assumed (which is reasonable, IMO)

Reply:
Well, understand the text, but it reads as if was written immediately after the first paragraph and then a lot of text was thrown in, end then "that protocol" stands dangling becuse it might mean either HTTP/1.1 or QUIC until you realize it cannot mean QUIC since QUIC didn't exist when HTTP/2 was written. So I merely suggest typing out HTTP/1.1 explicitly.

Also, my feedback have come across a bit grumpy, that was not the intention. I really do appreciate the extra clarifying text on this topic.

Reply:
Actually I am missing that. I was assuming HTTP/1.1 but now it is HTTP/3. So yes, I think a clarification is needed.